viewer/src/views/threejs/cad/CadNode.ts

```diff
@@ -160,6 +162,9 @@ export class CadNode extends THREE.Object3D {
   }
 
   private createLoadSectorsPipeline(): Subject<ThreeCameraConfig> {
+    const loadSectorOperator = flatMap((s: WantedSector) => this._repository.loadSector(s));
+    const consumeSectorOperator = flatMap((sector: ParsedSector) => this.rootSector.consumeSector(sector.id, sector));
+
```

Comment:
These changes are pretty much reversing the changes I made after feedback from Joachim. Some of his requests were to move the RxJS stuff all the way out into separate classes and functions and not just wrap regular functions with all the RxJS stuff left in CadNode.

Again, I am not really opposed to either approach; they are mostly a matter of style to me. But since we are now changing them back and forth, I think we should have a discussion about which we actually prefer :smile: